lib/diagnostic.rb

```diff
@@ -40,30 +53,34 @@
 
 class Animal
   def initialize
-      @sound = nil
+    @sound = nil
   end
 
   def say_hello
-      puts "I am a #{self.class.name}, and I go '#{@sound}'"
+    puts "I am a #{self.class.name}, and I go '#{@sound}'"
   end
 end
 
 class Cat < Animal
   def groom
-    puts "lick... lick..."
+    puts 'lick... lick...'
   end
 end
 
 class HouseCat < Cat
   def initialize
-    @sound = "meow"
+    @sound = 'meow'
   end
 end
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = "I am a Housecat, and I go 'meow'"
+# HouseCat.new initializes a new Housecat object with the @sound of 'meow'.
+# When we call say_hello, first the Cat class is checked for a say_hello method; it doesn't have one,
```

Comment:
First, the _HouseCat_ class is checked; then the _Cat_ class; then the _Animal_ class (and then, eventually, the _Object_ class).